go.mod

```diff
@@ -1,8 +0,0 @@
-module github.com/src-d/imports
```

Comment:
Let's keep it. I think we can still force a vendor mode without removing this file.

Reply:
I've added it back, but right now it will require to test/build it with:
```
GO111MODULE=off go test -v ./...
```

at least in my case.